manifest/run.go

```diff
@@ -238,23 +239,22 @@ func pruneSyncs(syncs []sync.Sync) []sync.Sync {
 	return pruned
 }
 
-func waitForContainer(container string, service Service) error {
-	i := 0
+func waitForContainer(container string, service Service, wait int) error {
+	waitTimeout := time.add(time.parseDuration("%ds", wait))
 
 	for {
 		host := containerHost(container, service.Networks)
-		i += 1
 
 		if host != "" {
 			return nil
 		}
 
-		// wait 10s max
-		if i > 100 {
-			return fmt.Errorf("%s failed to start within 10 seconds", container)
+		// wait for the container to start, default: 10 seconds
+		if time.After(waitTimeout) {
+			return fmt.Errorf("%s failed to start within %d seconds", container, wait)
 		}
 
-		time.Sleep(100 * time.Millisecond)
+		time.Sleep(time.parseDuration("1s"))
```

Comment:
It seemed like your `time.Sleep()` incrementally gets longer and longer. I am not sure if that was the intention. I decided to just sleep for one second.